### DISCUSSION TITLE
Shape of OR gates

### DISCUSSION THREAD

**Discussion by zhijieshi** [logisim-evolution/logisim-evolution#1550] [General]
I do not really like the shape of OR gates. Is there an easy way to display OR gates in the shape shown in the logo? Thanks. 

![image](https://user-images.githubusercontent.com/45037924/193454391-b0b61b87-264c-4162-8656-a96b4d4fc7f7.png)

**Comment by BFH-ktt1** [comment]
To get this shape, you have to rewrite the drawing routine of the OR-gates, a PR in this direction is always welcomed.
